src/qmapshack/gis/db/CSetupWorkspace.cpp

```diff
@@ -36,6 +36,7 @@ CSetupWorkspace::CSetupWorkspace(CGisWorkspace* workspace, QWidget* parent)
     checkDbUpdate->setChecked(cfg.value("listenUpdate", false).toBool());
     linePort->setText(cfg.value("port", "34123").toString());
     checkDeviceSupport->setChecked(cfg.value("device support", true).toBool());
+    checkRouteConversion->setChecked(cfg.value("saveSubPoints", false).toBool());
```

Comment:
This is not the right place nor the right way. Track to route conversion has nothing to do with the workspace. Please use dialog to ask for that option. Within that dialog the user can change the name, too. And you could add some point limit and apply the Douglas Peuker algorithm to reduce points.

Reply:
I have pushed a new version with a new track to route dialog. This dialog combines setting/editing the route name, selecting a new project and sub-point conversion.

I will have a look at the Douglas Peuker algorithm and see if I could incorporate this algorithm also in the dialog.